Source/Core/Common/ArmCPUDetect.cpp

```diff
@@ -8,13 +8,15 @@
 #include <string>
 #include <thread>
 
+#ifndef __APPLE__
 #ifndef _WIN32
 #ifndef __FreeBSD__
```

Comment:
These three #ifndef could be collapsed to a single line, couldn't they?

Reply:
The __FreeBSD__ one can't be collapsed without changing the logic, but I can collapse the __APPLE__ and _WIN32 ones.